pyabc/sampler/redis_eps/work.py

```diff
@@ -130,12 +129,16 @@ def get_int(var: str):
 
         # check if in look-ahead mode and should sleep
         if is_look_ahead and get_int(N_EVAL) >= max_n_eval_look_ahead:
-            # sleep 0.1 seconds
+            # sleep ... seconds
```

Comment:
why ... when the 0.1 is still hard-coded?

Reply:
just for fun :) just wanted to indicate it's seconds, and when changing the 0.1 at some point not forget to change it twice